### PR TITLE
{Pipeline} print stacktrace when exception

### PR DESCRIPTION
**Pull request by qwordy**
I met an error in CI checks. I cannot reproduce it in my machine. So I edit pipeline in order to show the detailed error.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: scripts/ci/index_ref_doc.py [hunk 1]
```diff
@@ -57,7 +58,11 @@ def test(self):
                     os.mkdir(ref_doc_out_dir)
                 script_args = [sys.executable, REF_GEN_SCRIPT, '--extension-file', ext_file, '--output-dir',
                                ref_doc_out_dir]
-                check_call(script_args)
+                try:
+                    check_call(script_args)
+                except CalledProcessError as e:
```
Comment (yungezz) [inline scripts/ci/index_ref_doc.py:63]: why speific this error? what about other errors?

**Comment by haroldrandom** [comment]
It didnt' trigger the desired job runs on alertsmanagement extension.

**Comment by qwordy** [comment]
> 
> 
> It didnt' trigger the desired job runs on alertsmanagement extension.

I am modifying pipeline definition to show the error. It does not aim to trigger the failed doc job.

File: scripts/ci/index_ref_doc.py (hunk 1 above)
Comment (qwordy) [reply scripts/ci/index_ref_doc.py:63]: It raises this kind of error when it fails according to the doc. It's not recommended to catch a too-broad exception.

**Comment by haroldrandom** [comment]
> > It didnt' trigger the desired job runs on alertsmanagement extension.
> 
> I am modifying pipeline definition to show the error. It does not aim to trigger the failed doc job.

This job verify extension ref should include alertsmanagement like other extensions, but missing:
image

image

**Pull request merged**